client.go

```diff
@@ -221,7 +221,10 @@ func (c *Client) Reconnect() error {
 	}
 	c.connMu.Lock()
 	defer c.connMu.Unlock()
-	c.conn.Close()
+	err := c.conn.Close()
```

Comment:
If there is something wrong with the connection, the Close might return error. And it is impossible to reconnect if we just return error, right? 

I think we should just log it and ignore error .

Reply:
We don't typically log directly from the client package.

Reply:
What is the purpose of capturing the close error? Is it track a leaked resource or help explain a connection error?

Reply:
It doesn't help, all right